src/dos/dos_mscdex.cpp

```diff
@@ -1178,8 +1196,25 @@ static Bitu MSCDEX_Interrupt_Handler(void) {
 							mscdex->ReadSectorsMSF(subUnit,raw,start,len,buffer);
 						break;
 					  };
-		case 0x83	:	// Seek - dont care :)
+		case 0x83	: { // Seek
+						uint8_t addressing_mode = mem_readb(curReqheaderPtr + 0x0D);
+						uint32_t sector = mem_readd(curReqheaderPtr + 0x14);
+						if (addressing_mode != 0) {
+							TMSF msf = {};
+							msf.min = (sector >> 16) & 0xFF;
+							msf.sec = (sector >> 8) & 0xFF;
+							msf.fr = (sector >> 0) & 0xFF;
+							sector = msf_to_frames(msf);
+							if (sector < REDBOOK_FRAME_PADDING) {
+								MSCDEX_LOG("MSCDEX: Seek: invalid position %d:%d:%d", msf.min, msf.sec, msf.fr);
+								sector = 0;
+							} else {
+								sector -= REDBOOK_FRAME_PADDING;
+							}
+						}
+						mscdex->Seek(subUnit, sector);
```

Comment:
Sending this through clang-format yields comical results, so maybe just leave it as is, yeah 😄